final report.adoc

```diff
@@ -0,0 +1,164 @@
+
+=== Budget
+
+We came up with the initial budget based on our estimations.
+
+When we knew the good news that our proposal was selected for GSoD 2021, we were all very excited. The team members promoted our GSoD project on the Internet and in universities. We ended up with a large number of applications from candidates around the world. After several discussions within the team, we decided to not just recruit one, but two technical writers for the project. The additional funding needed for the second writer was matched by the Metanorma project itself.
+
+=== Participants
+
+The core team working on this project was:
+
+* @kickoke and @techtrailhead (technical writer)
+* @ronaldtse, @opoudjis, @strogonoff (Technical mentors)
+* Scott Simmons and Marc Petit-Huguenin (Subject-matter mentors)
+
+Numerous Metanorma team members and Metanorma experts contributed to the GSoD team meetings and interviews:
+
+* @CAMOBAP
+* @Intelligent2013
+* @andrew2net
+* @ribose-jeffreylau
+* @manuel489
+* @anermina
+* @wkwong-ribose
+
+=== Timeline
+
+The planned project duration is 3 months. The progress is tracked on https://github.com/metanorma/gsod[Metanorma GitHub page].
+
+[cols="1,1,1"]
+|===
+|Project phase|Tasks|Schedule
+
+.4+|Analysis
+|Quantitative content audit
+|June 6th
+
+|Target audience analysis
+|June 13th
+
+
+|Qualitative content audit
+|June 13th
+
+|Create analysis report
+|June 15th
+
+.3+|Information architecture
+|Create content models and discuss reuse strategy
+|June 20th
+
+|Create new site structure for docs & navigation
+|27th June
+
+|Technical and editorial implementation of the new site structure
+|August 10th
+
+|Community
+|Set up a community platform
+|June 29th
+
+.2+|New Content
+|Create an FAQ page
+|Done initial work
+
+|Create a troubleshooting reference
+|July 20th
+
+.3+|Authoring Guide for New Users
+|Create an introductory tutorial
+|August 1st
+
+|Review
+|August 7th
+
+|Implement feedback
+|August 15th
+
+.4+|Flavor-specific Authoring guides
+|Create OGC Authoring guide
+|August 15th
+
+|Create ISO & IETF Authoring guide
+|August 15th
+
+|Review (Metanorma + SDOs)
+|August 10th
+
+|Implement feedback
+|August 15th
+
+|Wrap up
+|Create final GSoD report and roadmap for the future of docs
+|August 15th
+
+|===
+
+=== Results
+
+Details about the results can be found https://docs.google.com/presentation/d/1vLfafPX3gAakUwV8P5VMpqsGxupZ3WBkFGZY1J4V-fI/edit#slide=id.p1[here].
+
+The team carried out both quantitative audit and qualitative audit. 94 sources were analyzed (including website entries and slide decks). 
+Problems were identified and summarized in the https://github.com/metanorma/gsod/tree/master/artifacts/Analysis[analysis reports]. An information architecture was developed to tackle the problems.
+A tutorial for new users was created and some existing documentation was restructured.
+
```

Comment:
Comprehensive guides were created for ISO and OGC flavors along with cheat sheets for quick use by the users.

Reply:
Also IETF

Reply:
Added :)